tests/test_jsonchecker.py

```diff
@@ -38,6 +38,11 @@ async def test_check(self):
                         sha256="0000000000000000000000000000000000000000000000000000000000000000"
                     ),
                 )
+            elif data.filename == "jq-1.4.tarball_url.tar.gz":
+                self.assertNotEqual(
```

Comment:
Since we get specific release date from GH API, can we compare to it? I.e. use `assertEqual` instead of `assertNotEqual` here.